tests/tuner/test_scale_batch_size.py

```diff
@@ -123,7 +123,7 @@ def test_trainer_reset_correctly(tmpdir):
     assert actual == expected
 
 
-@RunIf(min_gpus=1)
```

Comment:
@carmocca we cannot run them like we do with GPUs. The issue is that CPU and GPU share memory here. So we cannot rely on anything like catching a CUDA OOM Error since when we run out of memory, the Python process will simply be killed by the OS from which we cannot recover.

Reply:
I rechecked all of the tests. the changes had to be pushed to the mps-accelerator PR though, since they do require the mps-logic. So this PR is really just renaming without any changes and the relevant changes will be coming together with the accelerator introduction

Reply:
Sounds good to me.

One last question I have is: Don't you think in the future we might want `RunIf(min_gpus=2)` to let tests run on both platforms? assuming both support what's being tested and we had CI jobs for each

Reply:
I would keep it separate. There are multiple reasons for that:

- Currently there is only 1 kind of device for each MPS machine, which makes with ddp etc. a bit difficult
- Most of the times, when the accelerator is specified, there are some device-checks included which were so far hardcoded for cuda, but that means, you either need to parametrize separately for the backends or you need to code the logic for that inside the test
- Not all ops are yet supported by MPS

That being said, I think it might make sense to unify cuda and rocm (once we have support for it) but due to the limitations both on hardware and backend side, I'd keep the mps backend separate for now